client-rest/examples/adsb-flow.rs

```diff
@@ -0,0 +1,113 @@
+//! Simulates a flow of ADS-B with multiple reporters
+
+use dotenv;
+use futures_lite::stream::StreamExt;
+use hyper::StatusCode;
+use hyper::{Body, Client, Method, Request};
+
+async fn mq_listener() -> Result<(), ()> {
+    let mq_addr = format!("amqp://localhost:5672");
```

Comment:
We should be able to deploy a rabbitmq cluster separately from the telemetry service.
The address should thus be configurable through the Environment, like we do for other dependencies.